resources/js/frontend/index.js

```diff
@@ -231,7 +268,6 @@ const Content = (props) => {
         checkboxSavedCardsComponent(labels.saveCardLabel)
     );
 };
-
 const Label = (props) => {
```

Comment:
@FabianaRBarros  It looks like it's not present in the `power_board` and in the branch `sync-main-with-power-board` this variable is not being used. So maybe it should be removed from your branch?

Reply:
![Screenshot 2024-09-04 at 15 30 54](https://github.com/user-attachments/assets/4d92041a-0dd7-4b3b-9522-774f49577228)